game_state.py

```diff
@@ -146,10 +146,23 @@ def get_pawn(self, row, col, color):
     def check_cell(self, row, col):
         return -1 if row < 0 or row > (self.width - 1) or col < 0 or col > (self.width - 1) else self.board[row][col]
 
-    def draw_board(self):
-        for i in range(3):
-            print(self.board[i])
-        print()
+    def draw_board(self):        
```

Comment:
I have aimed do this indentation everywhere, please highlight anywhere you've spotted I haven't

Reply:
Okay I see what you mean, it looks very out of place with only 2 spaces, at the time I wrote that section of code I was using a more basic IDE, since then I've switched to Visual Studio that follows the indentation formatting automatically for you. 
For my knowledge, do you know of anyway to check that the indentation is consistent automatically or is it a case of inspection by eye?

Reply:
Yes, the style guide that the majority of Python libraries follow is known as [PEP 8](https://pep8.org/). If you enjoy writing Python code, I suggest scanning through the examples occassionally to help remind yourself of what good style looks like.

To automate the process for consistent style, the tools that do this are known as "linters". See [Pylint](https://www.pylint.org/) as an example.